### PR TITLE
Pylint and RTD update patch, and other fixes

### PR DESCRIPTION
**Pull request by tekktrik**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ adafruit-circuitpython-simpleio
 adafruit-circuitpython-minimqtt
 adafruit-circuitpython-dps310
 adafruit-circuitpython-ahtx0
+sphinx>=4.0.0
```
Comment (kattni) [inline requirements.txt]: This needs to go into a new `/docs/requirements.txt` file, not in the `/requirements.txt` file. Please create the new file, add this to it, and remove it from here.

**Review by kattni — CHANGES_REQUESTED** [review]
Thanks! See below.

File: requirements.txt (hunk 1 above)
Comment (kattni) [reply requirements.txt]: @tekktrik I just realised you did it on all your PRs. I didn't notice it until now. Please go back through and submit another PR to fix this issue. Apologies for not being clearer about it in the beginning.

Comment (tekktrik) [reply requirements.txt]: Yeah, thank you for catching that, I'll go make those changes now

**Comment by tekktrik** [comment]
Okay, fixed the issue and submitted additional PRs for those repos that got approved.  Any that you haven't touched should also have the appropriate commit for this.  Thanks for catching it!

File: requirements.txt (hunk 1 above)
Comment (kattni) [reply requirements.txt]: @tekktrik I think this line still needs to be removed - unless the diff is acting up.

**Review by kattni — CHANGES_REQUESTED** [review]
One more thing. Please see above!

**Comment by tekktrik** [comment]
Got it, I'll check tonight an either change it or re-request the review if the diff is in fact acting up!

**Comment by tekktrik** [comment]
Yup, it was still there, just removed it!

**Review by kattni — APPROVED** [review]

**Pull request merged**